debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerHelper.java

```diff
@@ -277,13 +277,14 @@ static void startLogMining(Connection connection, Long startScn, Long endScn,
      * This method fetches the oldest SCN from online redo log files
      *
      * @param connection container level database connection
+     * @param archiveLogHours duration that archive logs are mined
      * @return oldest SCN from online redo log
      * @throws SQLException if anything unexpected happens
      */
-    static long getFirstOnlineLogScn(Connection connection) throws SQLException {
+    static long getFirstOnlineLogScn(Connection connection, Duration archiveLogHours) throws SQLException {
```

Comment:
```suggestion
    static long getFirstOnlineLogScn(Connection connection, Duration archiveLogDuration) throws SQLException {
```

Or maybe `archiveLogRetention`? No need to encode the unit of measurement in any case (here and elsewhere).